src/test/java/org/openrewrite/github/ActionsSetupJavaAdoptOpenJDKToTemurinTest.java

```diff
@@ -77,7 +77,7 @@ void actionsSetupJavaAdoptOpenJDKToTemurin() {
                     - name: set-up-jdk-2
                       uses: actions/setup-java@v2.3.0
                       with:
-                        distribution: "temurin"
+                        distribution: "adopt-openj9"
                         java-version: "11"
                     - name: build
                       run: ./gradlew build test
```

Comment:
This should not happen in this recipe

Reply:
@yeikel seems like this should still be temurin? Why the change?

Reply:
From the official migration docs 

> It is highly recommended to migrate workflows from adopt and adopt-openj9, to temurin and semeru respectively

It is not appropriate for the Temurin migration recipe to migrate `adopt-openj9` to `temurin` as the equivalent is the `semeru` distribution (what this new recipe will effectively do))